yogstation/code/modules/mob/living/simple_animal/friendly/goats.dm

```diff
@@ -0,0 +1,347 @@
+	desc = "WHAT THE HELL IS THAT!"
+	icon = 'yogstation/icons/mob/goats/horror_goat.dmi'
+	icon_state = "horrorgoat"
+	icon_living = "horrorgoat"
+	icon_dead = "horrorgoat_dead"
+	gold_core_spawnable = NO_SPAWN
+	attack_sound = 'sound/hallucinations/growl1.ogg'
+	attacktext = "lacerates"
+
+/mob/living/simple_animal/hostile/retaliate/goat/inverted
+	name = "Inverted Goat"
+	desc = "This goat seems to be from the 4th Dimension."
+	icon = 'yogstation/icons/mob/goats/inverted_goat.dmi'
+	icon_state = "invertedgoat"
+	icon_living = "invertedgoat"
+	icon_dead = "invertedgoat_dead"
+	gold_core_spawnable = NO_SPAWN
+
+/mob/living/simple_animal/hostile/retaliate/goat/memory
+	name = "Goat of your Past"
+	desc = "It feels like you seen this goat before but you cant place where..."
+	icon = 'yogstation/icons/mob/goats/memory_goat.dmi'
+	icon_state = "memorygoat"
+	icon_living = "memorygoat"
+	icon_dead = "memorygoat_dead"
+	gold_core_spawnable = NO_SPAWN
+
+/mob/living/simple_animal/hostile/retaliate/goat/mirrored
+	name = "taoG derorriM"
+	desc = ".noitisopsid tnasaelp rieht rof nwonk toN"
+	icon = 'yogstation/icons/mob/goats/mirrored_goat.dmi'
+	icon_state = "mirroredgoat"
+	icon_living = "mirroredgoat"
+	icon_dead = "mirroredgoat_dead"
+	gold_core_spawnable = NO_SPAWN
+
+/mob/living/simple_animal/hostile/retaliate/goat/paper
+	name = "Paper Goat"
+	desc = "Careful not to get a papercut!"
+	icon = 'yogstation/icons/mob/goats/paper_goat.dmi'
+	icon_state = "papergoat"
+	icon_living = "papergoat"
+	icon_dead = "papergoat_dead"
+	gold_core_spawnable = NO_SPAWN
+	butcher_results = list(/obj/item/paper = 3)
+
+/mob/living/simple_animal/hostile/retaliate/goat/pixel
+	name = "Pixel Goat"
+	desc = "How pixelated can we get?!?"
+	icon = 'yogstation/icons/mob/goats/pixel_goat.dmi'
+	icon_state = "pixelgoat"
+	icon_living = "pixelgoat"
+	icon_dead = "pixelgoat_dead"
+	gold_core_spawnable = NO_SPAWN
+
+/mob/living/simple_animal/hostile/retaliate/goat/radioactive
+	name = "Radioactive Goat"
+	desc = "I would not get near this goat if I were you"
+	icon = 'yogstation/icons/mob/goats/radioactive_goat.dmi'
+	icon_state = "radioactivegoat"
+	icon_living = "radioactivegoat"
+	icon_dead = "radioactivegoat_dead"
+	gold_core_spawnable = NO_SPAWN
+	light_power = 5
+	light_range = 4
+	light_color = LIGHT_COLOR_GREEN
+
+/mob/living/simple_animal/hostile/retaliate/goat/radioactive/Life()
+	radiation_pulse(src, 200) // It gets stronker as time passes
+
+/mob/living/simple_animal/hostile/retaliate/goat/rainbow
+	name = "Rainbow Goat"
+	desc = "WHAT DOES IT MEANNNNNNN!"
+	icon = 'yogstation/icons/mob/goats/rainbow_goat.dmi'
+	icon_state = "rainbowgoat"
+	icon_living = "rainbowgoat"
+	icon_dead = "rainbowgoat_dead"
+	gold_core_spawnable = NO_SPAWN
+	butcher_results = list(/obj/item/reagent_containers/food/snacks/meat/slab = 3, /obj/item/storage/crayons = 1, /obj/item/clothing/head/yogs/goatpelt = 1)
+
+/mob/living/simple_animal/hostile/retaliate/goat/spiffles
+	name = "Cute Goat"
```

Comment:
Shouldn't he be named spiffles?

Reply:
You ca also get it from a crate so I rather make it have a generic name and have the station one be named Spiffles then have each one be named Spiffles if you get what im saying

Reply:
oh right, that's fair